### DISCUSSION TITLE
Task2 奖励发放🎉

### DISCUSSION THREAD

**Discussion by phoouze** [openbuildxyz/mina-zkapp-bootcamp#339] [Announcements]
[Task2](https://github.com/openbuildxyz/mina-zkapp-bootcamp/blob/main/task/task2.md) 的奖励已发放，请查看[奖励统计表](https://github.com/openbuildxyz/mina-zkapp-bootcamp/blob/main/reward/task2.md)🎉

没有收到奖励的同学请：

查看自己的 pr 是否被合并，有问题的 pr 请及时处理。
查看自己是否在 learn/username/username.md 中提交钱包地址，未提交钱包地址的请尽快提交。
是否按照[规则](https://github.com/openbuildxyz/mina-zkapp-bootcamp#%E4%BB%BB%E5%8A%A1%E6%8F%90%E4%BA%A4)提交。
只有在 Task2 任务时间内(12月2日前)提交才有奖励， 请确认。
请检查自己的提交内容和 pr，及时更新。漏发的同学会进行补发，有任何问题请在本讨论区进行讨论。

**Comment by weirdcola** [comment]
请帮忙看一下这个， 在有效期内提交了 task2 ，后来解决了 pr 的问题，但是一直没有合并，也没有派发奖励。 https://github.com/openbuildxyz/mina-zkapp-bootcamp/pull/271

**Reply by phoouze** [reply]
今天会统一处理
